libnetwork/network.go

```diff
@@ -71,6 +71,7 @@ type networkDBTable struct {
 }
 
 // IpamConf contains all the ipam related configurations for a network
+// TODO(aker): use proper net/* structs instead of string literals.
```

Comment:
If you're updating, can you add an empty line in between? ISTR GoDoc can hide "TODO" comments in the generated documentation as long as there's an empty comment-line in between, so;


```go
// IpamConf contains all the ipam related configurations for a network.
//
// TODO(aker): use proper net/* structs instead of string literals.
```


(LOL, also this struct should really be named `IPAMConf` (to be properly camelcase, but that's a different topic)